service/svc.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+ROOTPROJECTPATH="$(
+    cd -- "$(dirname "$0")/.." >/dev/null 2>&1
+    pwd -P
+)"
+
+function install {
+    path=$1
+    if [ -z "$1" ]; then
+        path="/lib/systemd/system"
+    fi
+
+    if [ -e "$path/threeal-bot.service" ]; then
+        echo "Service is already installed"
+        exit
```

Comment:
This line indicate failure, use non zero exit code, see [this](https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html).